packages/inline-banner/addon/templates/components/nucleus-inline-banner.hbs

```diff
@@ -1,9 +1,19 @@
 {{#if hasBlock}}
-  {{yield
-    (hash
-      close=(action "onCloseTip")
-    )
-  }}
+  <div class="nucleus-inline-banner__icon">
+    {{nucleus-icon name=_icon size="medium" variant=type}}
+  </div>
+  <div class="nucleus-inline-banner__content">
+    {{yield}}
+  </div>
+  {{#if isDismissible}}
+    <div class="nucleus-inline-banner__close">
+      {{nucleus-button
+        variant="text"
+        size="mini"
+        icon="nucleus-cross"
+        onClick=(action "onCloseTip")}}
+    </div>
```

Comment:
If you notice, you're repeating the code inside if/else of hasBlock. Can you refactor it so that the same code is not repeated? 

> Principle: D.R.Y Don't Repeat Yourself

Reply:
I"ll refactor this code